stdlib/http/src/main/java/org/ballerinalang/net/http/websocket/client/InitEndpoint.java

```diff
@@ -64,57 +59,51 @@
 )
 public class InitEndpoint {
 
+    private static final Logger logger = LoggerFactory.getLogger(InitEndpoint.class);
+    private static final String STATEMENT_FOR_BACK_OF_FACTOR = "The decay's value set for the configuration needs " +
```

Comment:
```suggestion
    private static final String STATEMENT_FOR_BACK_OF_FACTOR = "The decay's value set of the configuration needs " +
```
is saying "decay's value set" correct?